ko/news/_posts/2017-03-28-ruby-2-2-7-released.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: news_post
+title: "루비 2.2.7 릴리스"
+author: "usa"
+translator: "shia"
+date: 2017-03-28 11:00:00 +0000
+lang: ko
+---
+
+루비 2.2.7이 릴리스 되었습니다.
+
+이번 릴리스는 이전 릴리스의 버그 70개를 수정합니다.
+[ChangeLog](http://svn.ruby-lang.org/repos/ruby/tags/v2_2_7/ChangeLog)를
+확인해주세요.
+
+이번 릴리스를 마지막으로 루비 2.2의 일반 지원이 종료되며, 보안 지원 기간이
```

Comment:
I want to use consist term with previous posts.

https://github.com/ruby/www.ruby-lang.org/blob/master/ko/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md